### PR TITLE
[LIBS] Add stack info in ReferenceCount error

### PR DESCRIPTION
**Pull request by rvql**
**Phenomenon and reproduction steps**

Original debug info cannot trace to the real problem

**Root cause and solution**

Add call stack info to error message

**Impactions**

N/A

**Test method**

N/A

**Affected branch(es)**

* main

**Checklist**

- [ ] Dependencies update required
- [ ] Common bug (similar problem in other repo)

### CODE REVIEW

File: server/libs/pool/reference_count.go [hunk 1]
```diff
@@ -35,7 +36,7 @@ func (r *ReferenceCount) SubReferenceCount() bool {
 		return true
 	}
 	if *r != 0 {
-		log.Errorf("reference(%d) maybe double released", *r)
+		log.Errorf("reference(%d) maybe double released\n%s", *r, string(debug.Stack()))
```
Comment (sharang) [inline server/libs/pool/reference_count.go:39]: 有没有刷屏风险

Comment (sharang) [reply server/libs/pool/reference_count.go:39]: 是否需要 sleep 1s 并主动退出？
此时应该有严重bug

Comment (sharang) [reply server/libs/pool/reference_count.go:39]: 可能退出也不合适，有没有什么办法抑制 log count

Comment (rvql) [reply server/libs/pool/reference_count.go:39]: 会刷屏，其实原来也会刷

**Review by sharang — APPROVED** [review]

**Pull request merged**